docs/src/guide/extensions_cluster_autoscaler.md

```diff
@@ -13,6 +13,66 @@ The following parameters may be added on each pool definition to enable manageme
 * `min_size`: Define the minimum scale of a pool managed by the cluster autoscaler. Defaults to `size` when not provided.
 * `max_size`: Define the maximum scale of a pool managed by the cluster autoscaler. Defaults to `size` when not provided.
 
+The cluster-autoscaler will manage the size of the nodepools with the attribute `autoscale = true`. To avoid the conflict between the actual `size` of a nodepool and the `size` defined in the terraform configuration files, you can add the `ignore_size = true` attribute to the nodepool definition in the `worker_pools` variable. This parameter will allow terraform to ignore the [drift](https://developer.hashicorp.com/terraform/tutorials/state/resource-drift) of the size parameter for the specific nodepool.
+
+This setting is strongly recommended for nodepools configured with `autoscale = true`.
+
+Example:
+
+```
+worker_pools = {
+  np-autoscaled = {
+    description = "Node pool managed by cluster autoscaler",
+    size        = 2,
+    min_size    = 1,
+    max_size    = 3,
+    autoscale   = true,
+    ignore_size = true # allow nodepool size drift
```

Comment:
Is allowing drift the default? If yes, please indicate

Reply:
I think we need to be a bit more explicit and document the various scenarios in a page dedicated to cluster autoscaler.

Reply:
Come to think of it, `ignore_size` lends to confusion. Let's rename the variable to `ignore_initial_pool_size`.